### DISCUSSION TITLE
"inherited access" feature lacking basic controls and docs

### DISCUSSION THREAD

**Discussion by tonglil** [community/community#31412] [Actions]
> Inherit access from source repository (recommended) 
<img width="1035" alt="Screen Shot 2022-08-30 at 3 04 09 PM" src="https://user-images.githubusercontent.com/3250776/187552084-62273fee-1553-49f0-be53-0865b2e3ea78.png">

1. Unable to default to inherit access by default when publishing npm package.

2. Poor documentation about this feature, https://docs.github.com/en/packages/learn-github-packages/about-permissions-for-github-packages does not really describe what's going on here.

3. "un"-inheriting does not remove added members. Inheriting should be a synced value.

**Comment by Hobart2967** [comment]
+1

**Comment by jmteufel** [comment]
I also ran into this issue. The ecosystem around GitHub packages is lacking in some places. There's also [this bug](https://github.com/orgs/community/discussions/169223), that organization packages can not be listed through the API.

**Comment by dannysauer** [comment]
Inheriting also doesn't update when the repo updates. It's apparantly only synced when you check the box, and stays at that snapshot forever. 🤦‍♂️
